.gitignore

```diff
@@ -8,6 +8,7 @@
 /apache/app-c2corg_ui.wsgi
 /apache/wsgi.conf
 /MANIFEST
+/nbproject
```

Comment:
Thanks for the tip @gberaudo 
This means that this commit should be dropped (using ``git rebase -i`` as we did it for other tree changes).